src/main/java/com/auth0/client/mgmt/JobsEntity.java

```diff
@@ -105,4 +107,30 @@
         request.setBody(requestBody);
         return request;
     }
+
+    /**
+     * Requests a Users Imports job. A token with scope write:users is needed.
+     * See https://auth0.com/docs/api/management/v2#!/Jobs/post_users_imports.
+     * See https://auth0.com/docs/users/guides/bulk-user-imports.
+     *
+     * @param connectionId The id of the connection to import the users to.
+     * @param users        The users file. Must have an array with the users' information in JSON format.
+     * @return a Request to execute.
+     */
+    public Request<Job> importUsers(String connectionId, File users) {
+        Asserts.assertNotNull(connectionId, "connection id");
+        Asserts.assertNotNull(users, "users file");
+
+        String url = baseUrl
+                .newBuilder()
+                .addPathSegments("api/v2/jobs/users-imports")
+                .build()
+                .toString();
+        MultipartRequest<Job> request = new MultipartRequest<>(client, url, "POST", new TypeReference<Job>() {
+        });
+        request.addPart("connection_id", connectionId);
```

Comment:
Let's add an upsert boolean parameter and a string externalId parameters to this method and conditionally add them to the multipart request for flexibility/matching the .NET one.

Reply:
Added an options object with 3 optional props:
- send completion email
- upsert
- external_id

Reply:
@damieng The boolean props are not returned at any moment. Should we keep them on the `Job` POJO object then? 😕

Reply:
Nevermind... They are not part of the `Job` POJO today 👍

Reply:
Options added in d5390391986fb4fa7ff9a4a66afdc5d560431b55